runtimes/eoapi/stac/eoapi/stac/client.py

```diff
@@ -512,12 +512,17 @@ async def item_collection(
             )
 
         if output_type == MimeTypes.html:
+            offset = int(request.query_params.get("offset") or 0)
+            limit = int(request.query_params.get("limit") or 10)
```

Comment:
there is no `offset` for `/items` or `/search` but we use `token` 

both are available as `limit` and `token` variables (L453 and L460)

Reply:
:+1: made the change